doc/source/conf.py

```diff
@@ -308,3 +355,26 @@
 Shall you encounter any issues, please feel free to report them on the
 [PyWavelets issue tracker](https://github.com/PyWavelets/pywt/issues)."""
 )
+
+# -- Options for MyST-NB and Markdown-based content --------------------------
+
+os.environ["PYDEVD_DISABLE_FILE_VALIDATION"] = "1"
+
+nb_execution_mode = 'auto'
+nb_execution_timeout = 60
+nb_execution_allow_errors = False
+
+nb_render_markdown_format = "myst"
+render_markdown_format = "myst"
+
+nb_remove_code_source = False
+nb_remove_code_outputs = False
+
+myst_enable_extensions = [
+    'amsmath',
+    'colon_fence',
+    'dollarmath',
+]
+
+# nb_execution_allow_errors = True
+# nb_execution_show_tb = True
```

Comment:
I'm not sure what to do with these configuration values, probably best to remove them since both default to `True`?

```suggestion
```